### PR TITLE
Feature Envy Resolved.

### PR DESCRIPTION
**Pull request by MatthijsJWH**
Resolved the feature envy design problem,
Added inCode analysis result.
And resolved some checkstyle warnings.
<organisation, documentation.

### CODE REVIEW

**Comment by SunweiWang** [comment]
Nicely merged with master, and Feature Envy also fixed. 
< review

File: src/main/MainScreenController.java [hunk 1]
```diff
@@ -449,28 +458,154 @@ public static PlayerFish getPlayerFish() {
...
+            lancebb.setY(screenBox.getHeight() / 4 * 3);
+          }
+          setBossMode(true);
+        }
 
-	public static void setBossMode(boolean bossMode) {
-		MainScreenController.bossMode = bossMode;
-	}
+        if (currScore > 500 && !bomb1) {
+          playerFish.getBombs().add(FishBomb.createFishBomb(playerFish));
```
Comment (SunweiWang) [inline src/main/MainScreenController.java]: But now when I run inCode, it shows MainScreenController is a Schizophrenic Class now.
< review

**Comment by MatthijsJWH** [comment]
Damn it, I didn't have that when I run it before merging, can somebody please help and check if we can fix it together, because we don't have that much time
< review

**Comment by MatthijsJWH** [comment]
In the worst case we can always choose to ignore it, but then we probably need to have a good reason to do so.
< review

**Comment by Michieldoesburg** [comment]
I can fix it.

**Comment by Michieldoesburg** [comment]
I cant fix it.

**Comment by MatthijsJWH** [comment]
I haven't found a fix either.

**Comment by SunweiWang** [comment]
Okay, we are going to solve this in our next iteration.

**Comment by MatthijsJWH** [comment]
Yeah I think that is the best solution.

**Comment by Michieldoesburg** [comment]
Yeah the severity of this problem is just 1, which is a lot lower than the feature envy was ( 4 I believe? ). Refactoring mainscreencontroller is a big task which will take at least a whole sprint. 

So lets merge this.

**Pull request merged**